.github/workflows/validate-generated-files.yml

```diff
@@ -21,4 +21,5 @@ jobs:
           export PATH=$PATH:$(go env GOPATH)/bin
           make install-tools
           make generate
+          make buf
```

Comment:
These commands can be in the same line, make a b c d. Make will bail if one in the order fails